tune_sklearn/tune_basesearch.py

```diff
@@ -253,7 +289,10 @@ def __init__(self,
                  return_train_score=False,
                  local_dir="~/ray_results",
                  max_iters=1,
-                 use_gpu=False):
+                 use_gpu=False,
+                 loggers=None,
+                 logdir="./",
+                 logconfig={}):
```

Comment:
be careful - https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments